Library/ViewModels/NoShippingPledgeViewModel.swift

```diff
@@ -94,19 +94,19 @@ public class NoShippingPledgeViewModel: NoShippingPledgeViewModelType, NoShippin
 
     let backing = project.map { $0.personalization.backing }.skipNil()
     self.showPledgeOverTimeUI = project.signal
-      .map { _ in featurePledgeOverTimeEnabled() }
+      .map { ($0.isPledgeOverTimeAllowed ?? false) && featurePledgeOverTimeEnabled()
+      }
 
     self.pledgeOverTimeConfigData = self.showPledgeOverTimeUI
+      .filter { $0 }
```

Comment:
nit: personally, I'd prefer to make this more explicit like `.filter { showUI in showUI == true }` but that's a minor nit.